cmd/createGoogleEnvironment.go

```diff
@@ -0,0 +1,104 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/fugue/fugue-client/client/environments"
+	"github.com/fugue/fugue-client/format"
+	"github.com/fugue/fugue-client/models"
+	"github.com/spf13/cobra"
+)
+
+type createGoogleEnvironmentOptions struct {
+	Name                string
+	ServiceAccountEmail string
+	ProjectID           string
+	ScanInterval        int64
+	ComplianceFamilies  []string
+}
+
+// NewCreateGoogleEnvironmentCommand returns a command that creates an environment
+func NewCreateGoogleEnvironmentCommand() *cobra.Command {
+
+	var opts createGoogleEnvironmentOptions
+
+	cmd := &cobra.Command{
+		Use:     "environment",
+		Short:   "Create an Google environment",
+		Aliases: []string{"env"},
+		Run: func(cmd *cobra.Command, args []string) {
+
+			client, auth := getClient()
+
+			scanScheduleEnabled := opts.ScanInterval != 0
+			var scanIntervalPtr *int64
+			if scanScheduleEnabled {
+				scanIntervalPtr = &opts.ScanInterval
+			}
+
+			params := environments.NewCreateEnvironmentParams()
+			params.Environment = &models.CreateEnvironmentInput{
+				ComplianceFamilies:     opts.ComplianceFamilies,
+				Name:                   opts.Name,
+				Provider:               "google",
+				ScanInterval:           scanIntervalPtr,
+				SurveyResourceTypes:    []string{},
+				RemediateResourceTypes: []string{},
+				ScanScheduleEnabled:    &scanScheduleEnabled,
+
+				ProviderOptions: &models.ProviderOptions{
+					Google: &models.ProviderOptionsGoogle{
+						ServiceAccountEmail: opts.ServiceAccountEmail,
+						ProjectID:           opts.ProjectID,
+					},
+				},
+			}
+
+			resp, err := client.Environments.CreateEnvironment(params, auth)
+			CheckErr(err)
+			env := resp.Payload
+
+			families := strings.Join(env.ComplianceFamilies, ",")
+
+			items := []interface{}{
+				Item{"ENVIRONMENT_ID", env.ID},
+				Item{"NAME", env.Name},
+				Item{"PROVIDER", env.Provider},
+				Item{"SCAN_INTERVAL", env.ScanInterval},
+				Item{"LAST_SCAN_AT", format.Unix(env.LastScanAt)},
+				Item{"NEXT_SCAN_AT", format.Unix(env.NextScanAt)},
+				Item{"SCAN_STATUS", env.ScanStatus},
+				Item{"COMPLIANCE_FAMILIES", families},
+				Item{"PROJECT_ID", env.ProviderOptions.Google.ProjectID},
```

Comment:
The AWS / Azure commands don't return any of the environment specific metadata in this table. This seems like a good change - should we propagate the concept out to those create/update commands?

Reply:
This is the create... What do you mean by propagating this to "create/update commands"?
The update has already a:
```
			case "google":
				items = append(items, Item{"PROJECT_ID", env.ProviderOptions.Google.ProjectID})
			}
```